packages/ui-components/src/components/QrCode/QrCodeVisual.tsx

```diff
@@ -92,58 +100,52 @@ export const QrCodeVisual = ({
   if (!isEnlarged) {
     if (options.length > 1) {
       return (
-        <Container className={className} maxWidth="sm">
-          <p style={{ color: 'white', textAlign: 'center' }}>QR Code</p>
+        <StyledContainer className={className} maxWidth="sm">
+          <StyledParagraph>QR Codes</StyledParagraph>
           {options
-            .filter((option, index) => index < 6)
+            .filter((option, index) => index < 4)
             .map(({ name, value }) => (
-              <Grid container style={{ paddingBottom: 5 }} alignContent="center">
-                <Grid item alignItems="flex-end" xs={6}>
-                  <QrCodeImage
-                    qrCodeContents={value}
-                    humanReadableId={name}
-                    width={50}
-                    margin="auto"
-                  />
-                </Grid>
-                <Grid item alignItems="flex-start" xs={6} style={{ justifyItems: 'center' }}>
-                  <Typography variant="body2" style={{ color: 'white' }}>
-                    {name}
-                  </Typography>
-                </Grid>
+              <Grid container direction="column">
+                <QrCodeImage
+                  qrCodeContents={value}
+                  humanReadableId={name}
+                  width={200}
+                  margin="auto"
+                  padding="0.5rem 0"
```

Comment:
I definitely think it would be better to take care of the margin, width, and padding in here instead of passing them as props. Seems like an anti-pattern to pass them through